### DISCUSSION TITLE
Can't play let's go Pikachu

### DISCUSSION THREAD

**Discussion by MNBY16** [skyline-emu/skyline#1596] [Q&A]
i open let's go Pikachu, the game shows a black screen, I can only see the buttons, but not the game content, what should I do?

**Comment by 13gkd** [comment]
Apply this settings
![Screenshot_2022-06-21-09-41-27-090_skyline emu](https://user-images.githubusercontent.com/103353628/174714617-6f580a6e-86aa-4139-bfa6-d414f34b037d.jpg)

**Comment by KYM201** [comment]
Exacto por favor hacer funcionar Pokémon lets go Pikachu plis 🙏🙏🙏🙏🙏🙏
